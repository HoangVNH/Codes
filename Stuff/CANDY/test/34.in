10
29
VXXVVVVVDDDDDVVVVVVVVVVXDXXXX
34
VVVVVVVVVVVVVDDDDDXXXXXVVVVVVVVVVD
47
VVXXXXXXXXXDDXXDDDXXXXXXXVVVDDDDVVVVXXXXXDDDDDD
50
VVVVXXXVVVVXXXXXXXXXXXXVVVVVVXXDDXXXXXXXVVVVVVVVVV
21
VVXXDVVVVVXXDDDDDDDDD
28
DXXVVVVVVVVVVVVVVDDDDDVVVVXX
32
DDVVVVXXXVVVVVVVVVVXXXXXVVVXXDDD
21
VVVVDDDDDDDDDXXDVVXXX
31
XXDDDDXXXXXXXVVVVVVXXXXXXXDDDDD
47
XXXXXXXXVVVVVVVVVVVVVVVVVVVVVDVVVVVDDDDDDDVVVVX
