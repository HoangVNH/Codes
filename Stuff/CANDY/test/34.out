28
33
45
48
20
26
31
20
30
46
